src/graphql/root/mutation/user-update-username.ts

```diff
@@ -15,14 +16,16 @@ const UserUpdateUsernameMutation = GT.Field({
   args: {
     input: { type: GT.NonNull(UserUpdateUsernameInput) },
   },
-  resolve: async (_, args, { wallet }) => {
+  deprecationReason:
+    "Username will be moved to @Handle in Accounts. Also SetUsername should be used instead of UpdateUpdate to reflect the idempotenty of Handles",
```

Comment:
```suggestion
    "Username will be moved to @Handle in Accounts. Also SetUsername should be used instead of UpdateUsername to reflect the idempotency of Handles",
```